docs/pages/getting-started/developers.md

```diff
@@ -124,7 +124,7 @@ You can also email us directly at uswebdesignstandards@gsa.gov.
 * Uses a **[modified BEM](https://pages.18f.gov/frontend/css-coding-styleguide/naming/)** approach created by 18F for naming CSS selectors. Objects in CSS are separated by single dashes. Multi-word objects are separated by an underscore (For example: `.usa-button-cool_feature-active`).
 * Uses **modular CSS** for scalable, modular, and flexible code.
 * Uses **nesting** when appropriate. Nest minimally with up to two levels of nesting.
-* **Global variables** are defined in the `_defaults.scss` file. Custom theming can be done by copying `_variables.scss`, changing variable values and importing it before uswds in Sass.
+* **Global variables** are defined in the `_defaults.scss` file. Custom theming can be done by copying `_variables.scss` into your own project's Sass folder, changing variable values, and importing it before uswds in Sass.
```

Comment:
@maya Would it make sense to say "changing the relevant variable values," or is that overkill?